addons/project_todo/static/src/scss/todo.scss

```diff
@@ -66,3 +80,30 @@
 .o_todo_hide_avatar .o_avatar {
     display: none;
 }
+
+.o_todo_chatter {
+    .o_FormRenderer_chatterContainer {
+        padding: 0;
+    }
+    li.nav-item {
+        display: none;
+    }
+}
+
+.o_todo_tags {
+    @include media-breakpoint-up(md) {
+        overflow: auto;
+    }
+}
+
+.o_todo_control_panel, .o_todo_panel_field {
+    @include media-breakpoint-up(md) {
+        overflow: auto;
+    }
+}
+
+.o_todo_chatter {
+    @include media-breakpoint-up(md) {
+        max-width: 30%;
+    }
```

Comment:
Is it not feasible to achieve this using Bootstrap classes because they are precompiled?